### PR TITLE
Format all integration and testutils Python code with black formatter

### PR DESCRIPTION
**Pull request by lluiscampos**
Format all integration and testutils Python code with black formatter and start enforcing it in the CI Pipeline.

Even though it is still running with Python 2, it should be okey to
use Python 3 style.

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -1,7 +1,15 @@
...
-include = '.*/backend-tests/.*\.py$'
+include = '''
+(
+  /(
+      testutils/.*\.py$
+    | backend-tests/.*\.py$
+    | tests/.*\.py$
+  )/
+)
+'''
```
Comment (lluiscampos) [inline pyproject.toml]: @oleorhagen Please pay close attention to this regex. I have low confidence in that I got it right

Comment (oleorhagen) [reply pyproject.toml]: Not completely right I am afraid, so good instincts :)
Try this:
```python
include = '''
(
  /(
      backend-tests
    | testutils
    | tests
  )/.*\.py$
)
```

Comment (lluiscampos) [reply pyproject.toml]: That works, thanks!

**Comment by lluiscampos** [comment]
> I'm gonna approve in spite of the comment, just remember to fix it :)

I am pushing now the amendment and I will be merging without running tests again, as they already passed in the previous push.

Thanks!

**Pull request merged**